pkg/helm/handlers/handlers.go

```diff
@@ -255,6 +256,9 @@ func (h *helmHandlers) HandleGetReleaseHistory(user *auth.User, w http.ResponseW
 }
 
 func (h *helmHandlers) HandleIndexFile(user *auth.User, w http.ResponseWriter, r *http.Request) {
+	// Setting this by default to true, this always serves helm index file with compatible chart lists.
+	onlyCompatible := true
```

Comment:
that should be default behaviour, but we should have a possibility to set this to `false`. We should have tests that verify that as well.

Reply:
done